slick/src/main/scala/slick/jdbc/GetResult.fm

```diff
@@ -3,8 +3,10 @@ package slick.jdbc
 import java.sql.{Date, Time, Timestamp}
 import slick.SlickException
 import slick.util.TupleSupport
+import scala.annotation.implicitNotFound
 
 /** Basic conversions for extracting values from PositionedResults. */
+@implicitNotFound("No GetResult found for type ${r"${T}"}. Try to implement an implicit GetResult for this type.")
```

Comment:
yep, other places use it because it adds info. I am doubtful this is a good place to use, it obscures the standard error message.